tools/generate-release-artifacts.sh

```diff
@@ -36,19 +43,22 @@ function step_complete() {
 }
 
 function build_binary_arm64() {
-  build_binary "arm64"
+  build_binary "linux" "arm64"
+  build_binary "darwin" "arm64"
```

Comment:
Added support to build artifacts also for `darwin` as well, because I wanted to test this changes on my laptop. Also made sense to have these artifacts along with `linux`.

If anyone have any historic context or conflict on these. Happy to know :)

Reply:
added comment in the next file, I would remove this line